src/main/resources/images/type/bug-ui.svg

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
```

Comment:
Inkscape added a lot of useless stuff, I suggest to give a quick round of [SVGOMG](https://jakearchibald.github.io/svgomg/) to reduce file sizes and/or to exclude those files from the generated JAR if they are not used.

Reply:
Good catch, I will remove the SVG